### ISSUE TITLE
android fullscreen and autoplay

### ISSUE THREAD

**Issue by sleith** [flowplayer/flowplayer#614]
Hi, i've been trying to make the video autoplay and full screen. But i don't have luck yet. Is it possible for flowplayer to do so on android?
Here's what i load with webview:

<html>

```
<head>

    <link
        href="file:///android_asset/skin/minimalist.css"
        rel="stylesheet"
        type="text/css" >

        <style type="text/css" >
```

body { font: 12px "Myriad Pro", "Lucida Grande", sans-serif; text-align: center; padding-top: 5%; }.flowplayer { width: 100%; height: 100%; }
            </style>

```
        <script
            src="http://ajax.googleapis.com/ajax/libs/jquery/1/jquery.min.js"
            type="text/javascript" >
        </script>

        <script
            src="file:///android_asset/flowplayer.min.js"
            type="text/javascript" >
        </script>
    </link>
</head>

<body>

    <div
        class="flowplayer"
        data-ratio="0.4167" >

        <video autoplay="autoplay" >

            <source
                height="100%"
                src="http://stream.flowplayer.org/bauhaus/624x260.mp4"
                type="video/mp4"
                width="100%" >
            </source>
        </video>
    </div>
</body>
```

</html>

Thanks
